classes/Context.php

```diff
@@ -370,16 +371,22 @@ public function getTranslator()
             return $this->translator;
         }
 
-        $translator = $this->getTranslatorFromLocale($this->language->locale);
-        $this->translator = $translator;
+        $sfContainer = SymfonyContainer::getInstance();
```

Comment:
You'll notice that I didn't use the **ContainerFinder**, the reason is that in its **getContainer** method, the symfony container comes last, so I ended up having another service container that doesn't have the translator service, even though the translator service is actually available via the Symfony container.

Reply:
That's weird, there shouldn't be more than one container in each environment. Maybe that's pointing to another bug somewhere else?

Reply:
Oh you meant in the front office? Then maybe we should declare it as a FO service, don't you think?

Reply:
No I mean in back office, the ContainerFinder gives me an instance of **AdminContainer** instead of giving me a **SymfonyContainer**...

Reply:
An issue has been created regarding this service container issue:

https://github.com/PrestaShop/PrestaShop/issues/14995